_posts/2015-02-23-integrating-with-poynt-cloud-apis.md

```diff
@@ -22,25 +22,47 @@ To consume Poynt APIs you must:
 1. Signup on [Poynt Developer Portal](https://poynt.net/auth/signup/developer)
 2. After you’re registered, you will see Poynt Developer dashboard.
     - All developers by default are assigned their default developer organization so they can invite other teammates to join and access the same applications. You can try this out later after setting up your device.
-3. On the dashboard, click on "COMPLETE YOUR PROFILE”, enter any missing information (eg. Website Url) and click on Save.
-4. On the dashboard, click on “CREATE A TEST MERCHANT” to create a test merchant for your development activity.
-![Create Cloud Application]({{site.url}}/developer/assets/poynt_net_create_new_app2.png)
-5. Once a test merchant is created, go to  "DEVELOPMENT" > "Cloud apps" and   click on "CREATE A CLOUD APPLICATION", to create an application to integrate with Poynt APIs.
-6. Fill in all the required fields and submit.<br/>
-![Create Cloud Application]({{site.url}}/developer/assets/poynt_net_new_app2.png){:height="800px" width="600px"}
-7. Save your newly created application credentials.<br/>
-![Create Cloud Application]({{site.url}}/developer/assets/poynt_net_app_created_credentials2.png){:height="350" width="600"}
-
-8. Obtain merchant's permission to access Poynt APIs on behalf of them (access their data)
+1. On the dashboard, click on "COMPLETE YOUR PROFILE”, enter any missing information (eg. Website Url) and click on Save.
+2. On the dashboard, click on “CREATE A TEST MERCHANT” to create a test merchant for your development activity. Select the Merchant Name and Bank and click "Submit"<br/>
+<img class="shadow"  src="{{site.url}}/developer/assets/poynt_net_create_test_merchant.png" alt="Create Test Merchant" height="500"/>
+
+5. Once a test merchant is created, go to  "DEVELOPMENT" -> click on "Create an app".<br/>
+<img class="shadow"  src="{{site.url}}/developer/assets/poynt_net_create_new_app.png" alt="Create New Application" height="500"/>
+
+6. Enter the App name and click on "Submit" button.<br/>
+<img class="shadow"  src="{{site.url}}/developer/assets/poynt_net_create_new_app2.png" alt="Create New Application" height="500"/>
+
+7. Save your newly created application credentials and click on "Go to app".<br/>
```

Comment:
Please call out that they need to click "Download keypair". There is a note further down in the page "Warning: Your private key cannot be recovered,...", please move it here.